lib/icinga/command.cpp

```diff
@@ -83,3 +86,14 @@ void Command::Validate(int types, const ValidationUtils& utils)
 		}
 	}
 }
+
+void Command::CommandChangeHandler(void)
```

Comment:
According to our super-new code formatting guide `(void)` should be just `()` (mostly to bring our code in line with what other projects appear to be doing, like LLVM).